.eslintrc

```diff
@@ -51,6 +52,7 @@ rules:
   no-var: 2
   object-curly-spacing: [2, never]
   one-var: [2, never]
+  operator-linebreak: [2, "after"]
```

Comment:
Could we do this: `[2, "after", { "overrides": { "?": "before", ":": "before"} }]` to promote this style:

```js
const foo = open
  ? 'open'
  : 'close';
```

this style is a lot cleaner for ternary, and a lot more readable when it's nested:

```js
const foo = halfOpen
  ? 'halfOpen'
  : open
    ? 'open'
    : 'close';
```